src/Unity.Mathematics/Noise/cellular2x2x2.cs

```diff
@@ -10,11 +10,12 @@ namespace Unity.Mathematics
 {
     public static partial class noise
     {
-        // Cellular noise, returning F1 and F2 in a float2.
-        // Speeded up by umath.sing 2x2x2 search window instead of 3x3x3,
-        // at the expense of some pattern artifacts.
-        // F2 is often wrong and has sharp discontinuities.
-        // If you need a good F2, use the slower 3x3x3 version.
+        /// <summary>
+        /// 3D Cellular noise ("Worley noise") with a 2x2x2 search window. Faster than using 3x3x3, at the expense of some pattern artifacts.
+        /// F2 is often wrong and has sharp discontinuities. If you need a smooth F2, use the slower 3x3x3 version.
+        /// </summary>
```

Comment:
Similar issue as `cellular2x2`, summary should be one liner and the rest in remarks.